### PR TITLE
Vietnamese translation: initial setup

### PR DESCRIPTION
**Pull request by daivinhtran**

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -70,6 +70,10 @@ translation:
   - changed-files:
       - any-glob-to-any-file: po/uk.po
 
+"translation/vn":
```
Comment (henrif75) [inline .github/labeler.yml:73]: Can you use the ISO code for Vietnamese and rename the .po file?

Comment (henrif75) [reply .github/labeler.yml:73]: https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=vi

File: po/vn.po
Comment (henrif75) [inline po/vn.po:1]: Please rename the file to vi.po
https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=vi

**Review by henrif75 — CHANGES_REQUESTED** [review]

**Comment by daivinhtran** [comment]
Thanks @henrif75 . I addressed your comment.

**Review by henrif75 — APPROVED** [review]

**Pull request merged**